erudit/templates/userspace/editor/form.html

```diff
@@ -8,6 +8,10 @@
 {% endblock %}
 
 {% block extrahead %}
+<script src="{% static 'vendor/jquery/dist/jquery.min.js' %}"></script>
```

Comment:
By adding this without adjusting pipeline config, we end up loading jquery twice. See also #96